package.json

```diff
@@ -13,7 +13,7 @@
 		"node": ">=6"
 	},
 	"scripts": {
-		"test": "xo && ava"
+		"test": "xo && tsd-check && ava"
```

Comment:
Wasn't sure about the order of execution.

Reply:
I would put it last. I don't want failing types to prevent me from running tests when developing.

Reply:
Ah, good point! Makes sense. Fixed it.